packages/core/src/interactions/packet/acknowledgement.ts

```diff
@@ -189,26 +191,28 @@ export class AcknowledgementInteraction {
       return
     }
 
-    if (!ticket.isWinningTicket(opening, response, ticket.winProb)) {
-      log(`Got a ticket that is not a win. Dropping ticket.`)
-      await this.db.markLosing(unacknowledged)
-      metric_losingTickets.increment()
-      return
-    }
-
-    // Ticket is a win, let's store it
+    // Store the acknowledged ticket, regardless if it's a win or a loss
+    // FIXME: preImage (`opening` here) should not be assigned at this moment.
     const ack = new AcknowledgedTicket(ticket, response, opening, unacknowledged.signer)
```

Comment:
I'd prefer to use a "place holder" preimage here, instead of changing the constructor. A proper fix would be remove the preImage field completely from the AcknowledgedTicket because they are only associated at the redemption. But changing that would result in a different data type stored in DB, not suitable for a hotfix.

Reply:
Changed